benchmarks/set_http_meta/config.yaml

```diff
@@ -0,0 +1,24 @@
+start: &defaults
```

Comment:
this and `no-useragentvariant` seem identical, we can probably just rename this one and remove the duplicate?

no strong opinion/no change is actually necessary, just thinking about how we should be comparing the results between these different configs.

Reply:
ok! I thought the "start" test works as warm up process, I remove it :+1:

Reply:
I thought that because some results look like:
```
sethttpmeta-start  |  31.5 us   26.8 | us: 1.17x faster   
sethttpmeta-no-useragentvariant | 26.6 us  | 27.1 us: 1.02x slower
```

Reply:
oh.... that might just be the variability of running it locally.

pyperf should handle the warm up per-scenario.